modules/Core/collections/panel/RecentRegistrations.php

```diff
@@ -60,8 +60,8 @@ public function getContent(): string {
                         'avatar' => $target_user->getAvatar(),
                         'uuid' => Output::getClean($item->uuid),
                         'groups' => $target_user->getAllGroupHtml(),
-                        'time' => $timeago->inWords(date('d M Y, H:i', $item->joined), $this->_language->getTimeLanguage()),
-                        'time_full' => date('d M Y, H:i', $item->joined)
+                        'time' => $timeago->inWords(date(DATE_FORMAT, $item->joined), $this->_language->getTimeLanguage()),
```

Comment:
This was probably not right initially, it might have worked but we could do with this being `'Y-m-d H:i:s'` since it's getting passed through to Timeago for further parsing